### PR TITLE
Added promotion types listing

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: examples/promotion-types.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        {
+            "id": "0dd46184-3c01-4ba0-8fa6-baf6e3abcd50",
+            "code": "PARFUM",
+            "name": "Parfum"
+        },
+        {
+            "id": "92d092c5-c023-49bb-b7f2-b90afde0e637",
+            "code": "SAISON",
+            "name": "Saison"
+        }
```
Comment (ivan-wolf) [inline examples/promotion-types.json:37]: There are types missing.

**Comment by ivan-wolf** [comment]
Link example in README.

File: examples/promotion-types.json (hunk 1 above)
Comment (ghost) [inline examples/promotion-types.json:37]: I used the same model as the defined under PromotionType:

``` swagger
PromotionType:
    type: object
    properties:
      id:
        $ref: '#/definitions/UUIDv4'
      code:
        type: string
        description: Navision promotion type code
      name:
        type: string
```

Isn't the represented by the code?

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ The Denner Portal provides mostly advertising related data.
 * `/article-groups` (Warengruppen, [example](examples/article-groups.json))
 * `/articles` (Artikel, [example](examples/articles.json))
 * `/advertised-articles` (beworbene Artikel)
-* `/promotions-types` (Aktionstypen)
+* `/promotion-types` (Aktionstypen)
```
Comment (ivan-wolf) [inline README.md]: The link to the example is still missing.

**Pull request merged**